src/_data/starters/11ty-tailscript.json

```diff
@@ -0,0 +1,7 @@
+{
+  "url": "https://github.com/4lador/11ty-tailscript",
+	"name": "11st-TailScript",
```

Comment:
@4lador Should this be `11st-` or `11ty-`?
(just noticing the filename wasn't matching the `name` here so thought I'd double check before merging)

Reply:
Hi, yes that's `11ty-` thanks for noticing :)